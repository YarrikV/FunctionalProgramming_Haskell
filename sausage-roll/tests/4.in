~ ~ ~ ~ ~ ~ ~ 
~ + +s+b+d+x~ 
~ + +n+ + +N~ 
~ ~ ~ ~ ~ ~ ~ 
