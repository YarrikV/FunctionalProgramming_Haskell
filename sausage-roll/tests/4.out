~ ~ ~ ~ ~ ~ ~ 
~ + +s+b+d+x~ 
~ + +n+ + +N~ 
~ ~ ~ ~ ~ ~ ~ 

~ ~ ~ ~ ~ ~ ~ 
~ +s+b+d+ + ~ 
~ +n+ + +x+W~ 
~ ~ ~ ~ ~ ~ ~ 

